polus-unet-testing-plugin/Dockerfile

```diff
@@ -0,0 +1,52 @@
+
+FROM labshare/polus-bfio-util:2.0.6-slim-buster
```

Comment:
Can you update this to `2.1.8`? That's the latest version of bfio. It's an Ubuntu container, so it shouldn't create any problems. If it does, just leave it as is and we can merge.